src/main/java/masecla/reddit4j/objects/RedditComment.java

```diff
@@ -0,0 +1,159 @@
+package masecla.reddit4j.objects;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class RedditComment extends RedditThing {
```

Comment:
Considering there's @Data, shouldn't these be private?

Reply:
They've now been made private